app/javascript/packages/verify-flow/forgot-password/index.tsx

```diff
@@ -0,0 +1 @@
+export { ForgotPassword } from './forgot-password';
```

Comment:
Can we consider this as part of the "password confirm" step and move the files into that folder? i.e.

```
app/javascript/packages/verify-flow/steps/password-confirm/forgot-password.tsx
```